core-services/importExport/src/main/java/org/silverpeas/core/importexport/wysiwyg/WysiwygContentType.java

```diff
@@ -24,18 +24,27 @@
 /*
  * Created on 1 mars 2005
  *
- * TODO To change the template for this generated file go to
- * Window - Preferences - Java - Code Style - Code Templates
  */
 package org.silverpeas.core.importexport.wysiwyg;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
 /**
  * @author sdevolder
  */
+@XmlRootElement(name = "wysiwyg")
+@XmlAccessorType(XmlAccessType.NONE)
 public class WysiwygContentType {
 
+  @XmlAttribute
   private String path;
 
+  public WysiwygContentType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)